### PR TITLE
docs: stacked bar worst-in-input and all option

### PR DESCRIPTION
**Pull request by rokamu623**
## Description
Added code and images for the `all` and `worst-in-input` options of the response time stacked bar.
Changed the image to be the same as the response time histogram image.

## Related links
Jira: https://tier4.atlassian.net/browse/RT2-862

## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.
- [x] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: docs/visualization/path/response_time.md [hunk 1]
```diff
@@ -74,15 +74,31 @@ plot.show()
...
 # plot best case
 plot = Plot.create_response_time_stacked_bar_plot(path, case='best')
 plot.show()
 ```
 
-![response_time_best_stacked_bar](../../imgs/response_time_best_stacked_bar.png)
+![response_time_best_stacked_bar](../imgs/response_time_stacked_bar_best.png)
+
+```python
+# plot best case
```
Comment (nabetetsu) [inline docs/visualization/path/response_time.md:88]:
```suggestion
# plot worst-in-input case
```

File: docs/visualization/path/response_time.md [hunk 2]
```diff
@@ -74,15 +74,31 @@ plot.show()
...
+plot = Plot.create_response_time_stacked_bar_plot(path, case='worst-in-input')
+plot.show()
+```
+
+![response_time_worst_in_input_stacked_bar](../imgs/response_time_stacked_bar_worst_in_input.png)
+
+```python
+# plot best case
```
Comment (nabetetsu) [inline docs/visualization/path/response_time.md:96]:
```suggestion
# plot all case
```

File: docs/visualization/path/response_time.md (hunk 1 above)
Comment (rokamu623) [reply docs/visualization/path/response_time.md]: Fixed it in [fb0f2c2](https://github.com/tier4/CARET_doc/pull/242/commits/fb0f2c266f386760dec191d05ed5980264018e89).

File: docs/visualization/path/response_time.md (hunk 2 above)
Comment (rokamu623) [reply docs/visualization/path/response_time.md]: Fixed it in [fb0f2c2](https://github.com/tier4/CARET_doc/pull/242/commits/fb0f2c266f386760dec191d05ed5980264018e89).

Comment (rokamu623) [reply docs/visualization/path/response_time.md]: Fixed it in [fb0f2c2](https://github.com/tier4/CARET_doc/pull/242/commits/fb0f2c266f386760dec191d05ed5980264018e89).

**Comment by rokamu623** [comment]
If one or more Emb teams approve, merge.

**Review by bopeng-saitama — APPROVED** [review]
LGTM

**Pull request merged**